.github/workflows/deploy.yml

```diff
@@ -40,6 +40,8 @@ jobs:
 
     - name: Publish package to PyPI
       uses: pypa/gh-action-pypi-publish@v1.10.0
```

Comment:
@nicoddemus so there is a bug in v1.10.0, and it explodes on attestation code path. It's fixed in v1.10.1 FYI.